client/app/components/Input/utils.js

```diff
@@ -57,7 +57,7 @@ export type Props = {
     | 'hidden'
     | 'switch'
     | 'location',
-  name?: string,
+  name: string,
```

Comment:
Could you explain why you made this a required value?

Reply:
It was the same problem as with nameValue. Keeping this property as optional resulted in having an error in the code of either DynamicForm or Form.

Reply:
If 'name' is marked as optional I get such error in the code:

![image](https://user-images.githubusercontent.com/25817917/76082421-a88bc500-5fab-11ea-94a3-b487d62f5250.png)